dask_cuda/proxify_host_file.py

```diff
@@ -164,6 +165,12 @@ def __init__(self, device_memory_limit: int, compatibility_mode: bool = None):
         else:
             self.compatibility_mode = compatibility_mode
 
+        # It is a bit hacky to forcefully capture the "distributed.worker" logger,
+        # eventually it would be better to have a different logger. For now this
+        # is ok, allowing users to read logs with client.get_worker_logs(), a
+        # proper solution would require changes to Distributed.
+        self.logger = logging.getLogger("distributed.worker")
```

Comment:
I'm not suggesting this needs to be done now, but we have [`LoggedBuffer`](https://github.com/rapidsai/dask-cuda/blob/4c175f4cf2ad86899efd52076f25f57cc4d7b830/dask_cuda/device_host_file.py#L25-L97) in `DeviceHostFile`, which gives us another lever of spill logging control.